### ISSUE TITLE
unknown device descriptor request failed

### ISSUE THREAD

**Issue by bamboo-master** [DeqingSun/ch55xduino#55]
I have 10 pcs. ch552 and ch551 of them 9 in the state "unknown device descriptor request failed" is it possible to somehow get out of this state?

**Comment by Kaitoukid93** [comment]
104 caps at 3v3

**Comment by bamboo-master** [comment]
Of course the capacitor is installed, that's not the reason.

**Comment by Kaitoukid93** [comment]
> 

can you show some pictures of the finished board( breadboard)?

**Comment by bamboo-master** [comment]
This is not a breadboard. I just soldered it into the adapter, although if you use breadboard, the result is the same.
![IMG_0831](https://user-images.githubusercontent.com/81521477/144705717-bc524b0a-c86c-45b0-9cbc-e6c683e48572.JPG)

**Comment by Kaitoukid93** [comment]
Why the resistor is pulling Tx to ground?, can you show the bottom of the board? If the chip is factory new, it will jump to bootloader mode regardless the state of USB-DPlus pin, if it's already programmed, you need to jump to bootloader mode by pulling USB-DP pin (pin 14) to +5v if you want to program the chip, or if you want to show as CDC device or HID device, use the example sketch

**Comment by bamboo-master** [comment]
It's not tx, it's usb d+ and it's just 12, not 14! see the datasheet, yes, I have it on 3.3v (in my opinion there is no difference), soldered to 5, the result is the same. From the very beginning, no COM ports appear, instead of them an "Unknown USB device", or is not detected at all. Another one seems to work, but after the firmware in the arduino ide, the "write dataflash failure" is no longer being flashed, another one worked until I unsealed the usb cable from it when I needed to flash it again, it stopped being determined again.
That's what's behind, although what can you see there? ordinary chinese sop/ssop16 to dip adapter
![IMG_0834](https://user-images.githubusercontent.com/81521477/144862901-21bccf6f-cbf1-48d0-a1bb-b48d1ff623fb.JPG)

**Comment by Kaitoukid93** [comment]
Ah yes, sorry, I read the pin number wrong
One more thing you can check is USB Dp, Dm, try to swap because in the picture (bottom) I can see the flat cabble being twisted, +5v__Dm__Dp__GND is the common order,if im not wrong,the wire next to 5v is connecting to Dp. Since I cant see the other end of the yellow flat cable, that's the best guess
I think this is just some wiring, soldering issues, I have order 2000 ch552G and none of them is dead

**Comment by bamboo-master** [comment]
Well, there is no point in changing usb d + d- it does not make sense at all if it is not correct, I tried to put a capacitor between them, too, it did not help. The problem may really be in the wired connection. But I don't know what to replace it with. It is not profitable for me to make a PCB.

**Comment by bamboo-master** [comment]
I found the time and began testing with a multimeter.
On one adapter on the reverse side where ssop16 between usb d+ and d- there was a "snot", I removed it with a soldering iron, but it did not get better, now the chip is simply not detected.
The second chip with the same diagnosis turned out to be just a "corpse", there is a short circuit between the ground and the power supply plus, this is without any adapters. The third chip is also a "corpse", it just warms up. For some reason, the built-in 3.3v stabilizer does not work on the fourth chip, which means that it is also a "corpse".
As a result, out of 10 pieces, 4 of them are not determined by the computer in any way, but the voltage stabilizer at 3.3v works, I would say "living corpses", plus one more is determined correctly, but is not recorded, 3 "corpses" and one still somehow works. It turns out on aliexpress it is better not to buy them at all! If you really want to take it in the form of "devboard" "devkit". Greedy sellers send obviously damaged chips, I took them from two different ones.
